AdminAPI.md

```diff
@@ -0,0 +1,90 @@
+# ACA-Py Administration API
+
+## Using the OpenAPI (Swagger) Interface
+
+ACA-Py provides an OpenAPI-documented REST interface for adminstering the agent's internal state and sparking communication with connected agents.
```

Comment:
*administering

Reply:
That's a typo from upstream, unfortunately lol